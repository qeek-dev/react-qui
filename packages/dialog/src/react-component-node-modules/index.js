import 'normalize.css'

export { default as TextDialog } from './components/Dialog/TextDialog'
export { default as PopupDialog } from './components/Dialog/PopupDialog'
export { default as DialogContainer } from './components/DialogContainer/DialogContainer'
