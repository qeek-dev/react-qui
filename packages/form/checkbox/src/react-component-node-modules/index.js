export { default as Checkbox } from './components/Checkbox/Checkbox'
export { default as CheckboxGroup } from './components/CheckboxGroup/CheckboxGroup'

