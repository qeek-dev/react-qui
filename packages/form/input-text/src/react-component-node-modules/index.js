export { default as InputText } from './components/InputText/InputText';
