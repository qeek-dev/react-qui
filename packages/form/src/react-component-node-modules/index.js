export { default as Checkbox } from './components/Checkbox/Checkbox'
export {
  default as CheckboxGroup,
} from './components/CheckboxGroup/CheckboxGroup'
export { default as Label } from './components/Label/Label'
export { default as InputText } from './components/InputText/InputText'
export { default as Qselect } from './components/Qselect/Qselect'
