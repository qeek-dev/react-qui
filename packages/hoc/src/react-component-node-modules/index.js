export { default as withDraggable } from './hoc/withDraggable';
