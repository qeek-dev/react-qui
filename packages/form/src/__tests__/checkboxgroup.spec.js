import React from 'react'
import ReactDOM from 'react-dom'
import { mount} from 'enzyme'
import { Checkbox, CheckboxGroup } from '../react-component-node-modules/index'
import { configure } from 'enzyme'
import Adapter from 'enzyme-adapter-react-16'
import 'jest-styled-components'

configure({ adapter: new Adapter() })

it('render <CheckboxGroup> component', () => {
  const checkboxGroup = mount(
    <CheckboxGroup direction="column">
      <Checkbox id="ethos" name="ethos" value="ethos">
        ETHOS
      </Checkbox>
      <Checkbox id="cvc" name="cvc" value="cvc">
        CVC
      </Checkbox>
      <Checkbox id="ink" name="ink" value="ink">
        INK
      </Checkbox>
    </CheckboxGroup>,
  )
  expect(checkboxGroup).toMatchSnapshot()
})

it('render CheckboxGroup correctly', () => {
  const checkboxGroup = mount(
    <CheckboxGroup direction="column">
      <Checkbox id="ethos" name="ethos" value="ethos">
        ETHOS
      </Checkbox>
      <Checkbox id="cvc" name="cvc" value="cvc">
        CVC
      </Checkbox>
      <Checkbox id="ink" name="ink" value="ink">
        INK
      </Checkbox>
    </CheckboxGroup>,
  )
  expect(checkboxGroup.children().text()).toEqual('ETHOSCVCINK')
})
