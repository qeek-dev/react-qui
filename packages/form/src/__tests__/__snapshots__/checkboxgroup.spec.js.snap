// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render <CheckboxGroup> component 1`] = `
.c1 {
  color: #000000;
  cursor: pointer;
  line-height: 0;
}

.c2 {
  margin-right: 0.625rem;
  vertical-align: middle;
}

.c3 {
  display: inline-block;
  vertical-align: middle;
}

.c4 {
  display: none;
}

.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-flow: column wrap;
  -ms-flex-flow: column wrap;
  flex-flow: column wrap;
  -webkit-box-pack: start;
  -webkit-justify-content: flex-start;
  -ms-flex-pack: start;
  justify-content: flex-start;
  -webkit-align-items: flex-start;
  -webkit-box-align: flex-start;
  -ms-flex-align: flex-start;
  align-items: flex-start;
}

.c0 > * {
  padding-bottom: 1.875rem;
  padding-right: 0rem;
}

<CheckboxGroup
  direction="column"
  span={30}
>
  <styled.div>
    <div
      className="c0"
    >
      <Checkbox
        checked={false}
        disabled={false}
        id="ethos"
        name="ethos"
        theme="light"
        value="ethos"
      >
        <div>
          <styled.label
            disabled={false}
            htmlFor="ethos"
            theme="light"
          >
            <label
              className="c1"
              disabled={false}
              htmlFor="ethos"
            >
              <styled.img
                alt="btn_checkbox"
                checked={false}
                disabled={false}
                src="btn_checkbox.svg"
                theme="light"
              >
                <img
                  alt="btn_checkbox"
                  checked={false}
                  className="c2"
                  disabled={false}
                  src="btn_checkbox.svg"
                />
              </styled.img>
              <styled.span>
                <span
                  className="c3"
                >
                  ETHOS
                </span>
              </styled.span>
            </label>
          </styled.label>
          <styled.input
            checked={false}
            disabled={false}
            id="ethos"
            name="ethos"
            onChange={[Function]}
            onClick={[Function]}
            value="ethos"
          >
            <input
              checked={false}
              className="c4"
              disabled={false}
              id="ethos"
              name="ethos"
              onChange={[Function]}
              onClick={[Function]}
              type="checkbox"
              value="ethos"
            />
          </styled.input>
        </div>
      </Checkbox>
      <Checkbox
        checked={false}
        disabled={false}
        id="cvc"
        name="cvc"
        theme="light"
        value="cvc"
      >
        <div>
          <styled.label
            disabled={false}
            htmlFor="cvc"
            theme="light"
          >
            <label
              className="c1"
              disabled={false}
              htmlFor="cvc"
            >
              <styled.img
                alt="btn_checkbox"
                checked={false}
                disabled={false}
                src="btn_checkbox.svg"
                theme="light"
              >
                <img
                  alt="btn_checkbox"
                  checked={false}
                  className="c2"
                  disabled={false}
                  src="btn_checkbox.svg"
                />
              </styled.img>
              <styled.span>
                <span
                  className="c3"
                >
                  CVC
                </span>
              </styled.span>
            </label>
          </styled.label>
          <styled.input
            checked={false}
            disabled={false}
            id="cvc"
            name="cvc"
            onChange={[Function]}
            onClick={[Function]}
            value="cvc"
          >
            <input
              checked={false}
              className="c4"
              disabled={false}
              id="cvc"
              name="cvc"
              onChange={[Function]}
              onClick={[Function]}
              type="checkbox"
              value="cvc"
            />
          </styled.input>
        </div>
      </Checkbox>
      <Checkbox
        checked={false}
        disabled={false}
        id="ink"
        name="ink"
        theme="light"
        value="ink"
      >
        <div>
          <styled.label
            disabled={false}
            htmlFor="ink"
            theme="light"
          >
            <label
              className="c1"
              disabled={false}
              htmlFor="ink"
            >
              <styled.img
                alt="btn_checkbox"
                checked={false}
                disabled={false}
                src="btn_checkbox.svg"
                theme="light"
              >
                <img
                  alt="btn_checkbox"
                  checked={false}
                  className="c2"
                  disabled={false}
                  src="btn_checkbox.svg"
                />
              </styled.img>
              <styled.span>
                <span
                  className="c3"
                >
                  INK
                </span>
              </styled.span>
            </label>
          </styled.label>
          <styled.input
            checked={false}
            disabled={false}
            id="ink"
            name="ink"
            onChange={[Function]}
            onClick={[Function]}
            value="ink"
          >
            <input
              checked={false}
              className="c4"
              disabled={false}
              id="ink"
              name="ink"
              onChange={[Function]}
              onClick={[Function]}
              type="checkbox"
              value="ink"
            />
          </styled.input>
        </div>
      </Checkbox>
    </div>
  </styled.div>
</CheckboxGroup>
`;
